### DISCUSSION TITLE
how to get it works for comments

### DISCUSSION THREAD

**Discussion by ChangrunHuang** [alefragnani/vscode-separators#103] [Q&A] [answered]
Hi,

ohh... I misunderstood its function...
I thought I can add separators for the comments I added to the code. Thanks.

-----------------------------------------------------------------------------------------------------------------------------
Thank you so much for this useful extension.
I notice in the newest release (ver 2.7), it said that it now supports drawing separators above comments.
However, I was not able to have it works.
<img width="1250" alt="image" src="https://github.com/user-attachments/assets/deb6a47e-6de4-4723-9275-b2cf3956f740">
Do I miss something in customazing to make it work?
thank you for your help
Changrun

**Comment by alefragnani** [comment, marked as answer]
Hi @ChangrunHuang ,

In order to make it draw the separators above the comments, you have to define the `separators.location` setting to `aboveTheComment`, like this:

```json
       "separators.location": "aboveTheComment"
```

As documented in the README https://github.com/alefragnani/vscode-separators#available-settings, you will notice the extension supports only _three comments patterns_, but it also allows you to create your own patter, if needed.

Hope this helps
